src/foam/u2/grid/seutil.js

```diff
@@ -0,0 +1,396 @@
+console.log("Loading SE.util");
```

Comment:
There's also foam.ENUM() and foam.RELATIONSHIP().
You should break up your library in smaller libraries of related functions.
src/foam/util is okay, we can always move it if we think of a better location.